<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OxygenStyleConfig</class>
 <widget class="QWidget" name="OxygenStyleConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>557</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="KTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="_toolBarDrawItemSeparator">
         <property name="text">
          <string>Draw toolbar item separators</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="_checkDrawX">
         <property name="text">
          <string>Use 'X' in checkboxes instead of check</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="_splitterProxyEnabled">
         <property name="text">
          <string>Enable extended resize handles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="_cacheEnabled">
         <property name="text">
          <string>Enable pixmap cache</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="_animationsEnabled">
         <property name="text">
          <string>Enable animations</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Windows' drag mode:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>_windowDragMode</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="KComboBox" name="_windowDragMode">
           <item>
            <property name="text">
             <string>Drag windows from titlebar only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Drag windows from titlebar, menubar and toolbars</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Drag windows from all empty areas</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="_mnemonicsLabel">
           <property name="text">
            <string>Keyboard accelerators visibility:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::HLine</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KComboBox" name="_mnemonicsMode">
           <item>
            <property name="text">
             <string>Always Hide Keyboard Accelerators</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Show Keyboard Accelerators When Needed</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Always Show Keyboard Accelerators</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="_useWMMoveResize">
         <property name="text">
          <string>Use window manager to perform windows' drag</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Views</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="_viewDrawFocusIndicator">
         <property name="text">
          <string>Draw focus indicator</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="_viewDrawTreeBranchLines">
         <property name="text">
          <string>Draw tree branch lines</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="_viewDrawTriangularExpander">
         <property name="text">
          <string>Use triangle tree expander instead of +/-</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>25</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QWidget" name="_viewsExpertWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Triangle size:</string>
            </property>
            <property name="buddy">
             <cstring>_viewTriangularExpanderSize</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KComboBox" name="_viewTriangularExpanderSize">
            <item>
             <property name="text">
              <string comment="triangle size">Tiny</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string comment="triangle size">Small</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string comment="triangle size">Normal</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>259</width>
           <height>70</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Tabs</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="_tabStyleSingle">
         <property name="text">
          <string comment="Renders each inactive tab in a tabbar as separate darker slab">Single</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="_tabStylePlain">
         <property name="text">
          <string comment="Renders inactive tabs in a tabbar as a unique darker slab">Plain</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Inactive tabs style:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Scrollbars</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Scrollbar width:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>_scrollBarWidthSlider</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="_scrollBarWidthSlider">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
           <property name="pageStep">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KIntSpinBox" name="_scrollBarWidth">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="suffix">
            <string>px</string>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Top arrow button type:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Bottom arrow button type:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KComboBox" name="_scrollBarSubLineButtons">
           <item>
            <property name="text">
             <string>No buttons</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>One button</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Two buttons</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="KComboBox" name="_scrollBarAddLineButtons">
           <item>
            <property name="text">
             <string>No button</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>One button</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Two buttons</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Menu Highlight</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QRadioButton" name="_menuHighlightDark">
         <property name="text">
          <string>Use dark color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="_menuHighlightStrong">
         <property name="text">
          <string>Use selection color (plain)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="_menuHighlightSubtle">
         <property name="text">
          <string>Use selection color (subtle)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="_expertModeContainer" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="_expertModeButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>_toolBarDrawItemSeparator</tabstop>
  <tabstop>_checkDrawX</tabstop>
  <tabstop>_cacheEnabled</tabstop>
  <tabstop>_animationsEnabled</tabstop>
  <tabstop>_mnemonicsMode</tabstop>
  <tabstop>_windowDragMode</tabstop>
  <tabstop>_useWMMoveResize</tabstop>
  <tabstop>_expertModeButton</tabstop>
  <tabstop>_viewDrawFocusIndicator</tabstop>
  <tabstop>_viewDrawTreeBranchLines</tabstop>
  <tabstop>_viewDrawTriangularExpander</tabstop>
  <tabstop>_viewTriangularExpanderSize</tabstop>
  <tabstop>_tabStyleSingle</tabstop>
  <tabstop>_tabStylePlain</tabstop>
  <tabstop>_scrollBarWidthSlider</tabstop>
  <tabstop>_scrollBarWidth</tabstop>
  <tabstop>_scrollBarSubLineButtons</tabstop>
  <tabstop>_scrollBarAddLineButtons</tabstop>
  <tabstop>_menuHighlightDark</tabstop>
  <tabstop>_menuHighlightStrong</tabstop>
  <tabstop>_menuHighlightSubtle</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_scrollBarWidth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_scrollBarWidthSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_scrollBarWidthSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_scrollBarWidth</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_viewDrawTriangularExpander</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_viewDrawTriangularExpander</sender>
   <signal>toggled(bool)</signal>
   <receiver>_viewTriangularExpanderSize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
